package.json

```diff
@@ -6,8 +6,9 @@
   "private": true,
   "license": "MPLv2",
   "scripts": {
-    "build": "npm-run-all clean build:extension",
+    "build": "npm-run-all clean build:extension build:finalize",
     "build:extension": "web-ext build -s ./extension/ --overwrite-dest",
+    "build:finalize": "mv web-ext-artifacts/*.zip web-ext-artifacts/remote-settings-devtools.xpi",
```

Comment:
this is required because we need the version number to not be in the filename or we cannot target the output file in the signing pipeline